usmqe_tests/api/gluster/test_gluster_cluster.py

```diff
@@ -186,8 +187,21 @@ def test_cluster_unmanage_valid(
         cluster_reuse["is_managed"] == "yes",
         "is_managed: {}\nThere should be ``yes``.".format(cluster_reuse["is_managed"]))
 
-    cluster_health = graphite_api.get_datapoints(
-        target="tendrl.clusters.{}.status".format(cluster_id))
+    # graphite target uses short name if it is set
+    if cluster_reuse["short_name"]:
+        cluster_target_id = cluster_reuse["short_name"]
+    else:
+        cluster_target_id = cluster_reuse["cluster_id"]
+    i = 0
+    while i < 31:
```

Comment:
This is nitpicking, but could you rather use something pythonic, like `for i in range(31):`